clang/runtime/dpct-rt/include/dpct/shmem_utils.hpp

```diff
@@ -0,0 +1,27 @@
+//==---- sparse_utils.hpp -------------------------*- C++ -*----------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __DPCT_SHMEM_UTILS_HPP__
+#define __DPCT_SHMEM_UTILS_HPP__
+
+namespace dpct::shmem {
+
+/// Signal Operation
```

Comment:
Please add doc for this function like sparse_utils.hpp

Reply:
Added the doc strings